main.nf

```diff
@@ -1157,6 +1161,32 @@ def checkParameterList(list, realList) {
   return list.every{ checkParameterExistence(it, realList) }
 }
 
+def checkParams(it) {
+ // Check if params is in this given list
+  return it in [
+  'callName',
+  'call-name',
```

Comment:
We do need both.
When I loop into params, to see every parameters, whenever there is a parameter with a uppercase letter, it adds another parameters with a - and a lower case letter to replace the first uppercase letter.
There is probably something to explain that, or maybe I didn't access it the right way...